README.md

```diff
@@ -1,2 +1,3 @@
 # code_review_practice
+
 This repository contains code to be reviewed by Insight fellows.
```

Comment:
Is there value in providing more detail?

Reply:
Nah, it's good as is for now.